### PR TITLE
Feature/hamburger menu

### PR DESCRIPTION
**Pull request by renatogalvones**
## Description
https://jirasoftware.catho.com.br/browse/QTM-91

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [ ] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [ ] Unit tests (yarn test:components)
- [ ] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [ ] Code review

### Browsers review
- [ ] Layout review
  - [ ] Edge
  - [ ] IE 11
  - [ ] Firefox
  - [ ] Chrome
  - [ ] Safari
  - [ ] Mobile
- [ ] Ux review validation

### CODE REVIEW

**Comment by rapahaeru** [comment]
its hard to compare, but i think this notification alert has a transparent border instead black.

Browser:
Captura de tela de 2019-10-31 10-27-30

zeplin:
Captura de tela de 2019-10-31 10-28-40

**Comment by rapahaeru** [comment]
## Review guide
- [x] Unit tests (yarn test:components)
- [x] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [ ] Code review

### Browsers review
- [ ] Layout review
  - [ ] Edge
  - [ ] IE 11
  - [ ] Firefox
  - [ ] Chrome
  - [ ] Safari
  - [ ] Mobile
- [ ] Ux review validation

File: components/Hamburger/Hamburger.jsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+};
+
+const HamburgerWrapper = styled.div`
+  display: inline-block;
+  position: relative;
+  width: 24px;
+  height: 24px;
+`;
+
+const NotificationIcon = styled.span`
```
Comment (pedrohmorais) [inline components/Hamburger/Hamburger.jsx]: What you think about change the notification border when the `inverted` prop is false ?

image

File: stories/Hamburger/Hamburger.regression-test.story.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      style={{
+        backgroundColor: '#000',
+        display: 'inline-block',
+      }}
+    >
+      <Hamburger showNotification />
+    </span>
+  ))
+  .add('isOpened', () => (
+    <span
```
Comment (pedrohmorais) [inline stories/Hamburger/Hamburger.regression-test.story.jsx]: What you think about to create a `styledSpan` component with `styled-components` lib instead to repeat this span with inline style ?

File: components/Hamburger/Hamburger.jsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+const getColors = ({ inverted }) => {
+  const weight = !inverted ? 0 : 900;
+
+  return colors.neutral[weight];
+};
+
+const HamburgerWrapper = styled.div`
+  display: inline-block;
+  position: relative;
+  width: 24px;
```
[the remark this answers is not in the document]
Comment (renatogalvones) [reply components/Hamburger/Hamburger.jsx]: Done.

File: components/Hamburger/Hamburger.jsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  height: 24px;
+`;
+
+const NotificationIcon = styled.span`
+  position: absolute;
+  width: 7px;
```
[the remark this answers is not in the document]
Comment (renatogalvones) [reply components/Hamburger/Hamburger.jsx]: done

File: stories/Hamburger/Hamburger.story.jsx [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+const description = `Hamburgers are used for showing quantity of something, as
+warnings, inbox messages and others.`;
+
+storiesOf('Hamburger', module).add('Hamburger', () => (
+  <AutoExample
+    description={description}
+    component={Hamburger}
+    componentProps={{
+      number: 10,
```
[the remark this answers is not in the document]
Comment (renatogalvones) [reply stories/Hamburger/Hamburger.story.jsx]: Nice catch!

File: stories/Hamburger/Hamburger.story.jsx [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+    <Hamburger isOpened />
+  </div>
+</div>`}
+        />
+      </Container>
+    </StoryContainer>
+  </Tab>
+);
+
+const description = `Hamburgers are used for showing quantity of something, as
```
[the remark this answers is not in the document]
Comment (renatogalvones) [reply stories/Hamburger/Hamburger.story.jsx]: Done

File: stories/Hamburger/Hamburger.regression-test.story.jsx (hunk 2 above)
Comment (renatogalvones) [reply stories/Hamburger/Hamburger.regression-test.story.jsx]: done

**Pull request merged**